---
title: Reconstruction of ancestral chromosome architecture and gene repertoire reveals
  principles of genome evolution in a model yeast genus
authors:
- Nikolaos Vakirlis
- Véronique Sarilar
- Guénola Drillon
- Aubin Fleiss
- Nicolas Agier
- Jean-Philippe Meyniel
- Lou Blanpain
- Alessandra Carbone
- Hugo Devillers
- Kenny Dubois
- Alexandre Gillet-Markowska
- Stéphane Graziani
- Nguyen Huu-Vang
- Marion Poirel
- Cyrielle Reisser
- Jonathan Schott
- Joseph Schacherer
- Ingrid Lafontaine
- Bertrand Llorente
- Cécile Neuvéglise
- Gilles Fischer
date: '2016-01-01'
publishDate: '2025-04-11T16:57:09.742073Z'
publication_types:
- article-journal
publication: '*Genome Research*'
doi: 10.1101/gr.204420.116
abstract: Reconstructing genome history is complex but necessary to reveal quantitative
  principles governing genome evolution. Such reconstruction requires recapitulating
  into a single evolutionary framework the evolution of genome architecture and gene
  repertoire. Here, we reconstructed the genome history of the genus Lachancea that
  appeared to cover a continuous evolutionary range from closely related to more diverged
  yeast species. Our approach integrated the generation of a high-quality genome data
  set; the development of AnChro, a new algorithm for reconstructing ancestral genome
  architecture; and a comprehensive analysis of gene repertoire evolution. We found
  that the ancestral genome of the genus Lachancea contained eight chromosomes and
  about 5173 protein-coding genes. Moreover, we characterized 24 horizontal gene transfers
  and 159 putative gene creation events that punctuated species diversification. We
  retraced all chromosomal rearrangements, including gene losses, gene duplications,
  chromosomal inversions and translocations at single gene resolution. Gene duplications
  outnumbered losses and balanced rearrangements with 1503, 929, and 423 events, respectively.
  Gene content variations between extant species are mainly driven by differential
  gene losses, while gene duplications remained globally constant in all lineages.
  Remarkably, we discovered that balanced chromosomal rearrangements could be responsible
  for up to 14% of all gene losses by disrupting genes at their breakpoints. Finally,
  we found that nonsynonymous substitutions reached fixation at a coordinated pace
  with chromosomal inversions, translocations, and duplications, but not deletions.
  Overall, we provide a granular view of genome evolution within an entire eukaryotic
  genus, linking gene content, chromosome rearrangements, and protein divergence into
  a single evolutionary framework.
---
